closed/custom/modules/java.base/Copy.gmk

```diff
@@ -50,8 +50,9 @@ $(call openj9_copy_files_and_debuginfos, \
 		$(OPENJ9_VM_BUILD_DIR)/ \
 		$(addprefix $(LIB_DST_DIR), / /j9vm/ /server/)))
 
-# CPU targets without JIT support.
-NO_JIT_CPUS := riscv64
+# target platforms without JIT support.
```

Comment:
Please punctuate consistently. Either capitalize the first word (my preference):
```
# Target platforms without JIT support.
```
or use a sentence fragment:
```
# target platforms without JIT support
```

Reply:
Updated based on the comment.

Reply:
This doesn't appear to have been changed.

Reply:
I fixed it.
It seems something went wrong when I updated my commit yesterday.